packages/vite/src/node/plugins/css.ts

```diff
@@ -108,7 +108,7 @@ export interface CSSModulesOptions {
     | null
 }
 
-const cssLangs = `\\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)($|\\?)`
+export const cssLangs = `\\.(css|less|sass|scss|styl|stylus|pcss|postcss|sss)($|\\?)`
```

Comment:
it'd be better to export `cssLangRE` on the line below instead of exporting this

Reply:
I would export `isCSSRequest`

Reply:
Oh yeah. That's even better. Let's do that